modules/services/lorri.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.lorri;
+  home = "${builtins.getEnv "HOME"}";
+in
+{
+  options =  {
+    services.lorri = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to enable the lorri service.";
+      };
+
+      logFile = mkOption {
+        type = types.nullOr types.path;
+        default = "${home}/Library/Logs/lorri.log";
+        example =  "${home}/Library/Logs/lorri.log";
+        description = ''
+          The logfile to use for the lorri service. Alternatively
+          <command>sudo launchctl debug system/com.target.lorri --stderr</command>
+          can be used to stream the logs to a shell after restarting the service with
+          <command>sudo launchctl kickstart -k system/com.target.lorri</command>.
+        '';
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ pkgs.lorri ];
```

Comment:
I only needed to add nix to make it work.

Reply:
As mentioned in that comment certain builtins depend on binaries indirectly. That's why things like git and tar are included in the nixos module, otherwise expressions that use those wouldn't work.

```
$ PATH=/nix/store/knw0knz4wkpdml3h3g2vb9fih2gvhfrx-nix-2.4pre20200721_ff314f1/bin nix-instantiate --eval -E 'builtins.fetchGit https://github.com/LnL7/nix-darwin.git'
error: error: --- SysError ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- nix-instantiate
executing 'git': No such file or directory
error: --- Error ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- nix-instantiate
program 'git' failed with exit code 1
```

Reply:
Ah now I get it. Thanks for the clarification!